schema-source/RSxxGlzSys.schema.yaml

```diff
@@ -0,0 +1,261 @@
+Schema:
+  Object Type: "Meta"
+  References:
+    - "ASHRAE205"
+    - "RSxxSub"
+  Title: "Fenestration - Glazing System"
+  Description: "Schema for ASHRAE 205 annex RSxxGlzSys: Glazing System"
+  Version: "2.0.0"
+  Root Data Group: "RSxxGlzSys"
+
+# Data Groups
+RSxxGlzSys:
+  Object Type: "Data Group"
+  Data Elements:
+    metadata:
+      Description: "Metadata data group"
+      Data Type: "{Metadata}"
+      Constraints: "schema=RSxxGlzSys"
+      Required: True
+    description:
+      Description: "Data group describing product and rating information"
+      Data Type: "{Description}"
+    performance:
+      Description: "Data group containing performance information"
+      Data Type: "{Performance}"
+      Required: True
+
+Description:
+  Object Type: "Data Group"
+  Data Elements:
+    product_information:
+      Description: "Data group describing product information"
+      Data Type: "{ProductInformation}"
+
+ProductInformation:
+  Object Type: "Data Group"
+  Data Elements:
```

Comment:
It should be possible to define the Universally Unique Identifier (UUID) of the component in the product data network buildingenvelopedata.org .

Reply:
Good idea